### ISSUE TITLE
Hydration fails on fresh app with "Expected server HTML to contain a matching <head> in <html>"

### ISSUE THREAD

**Issue by rohanliston** [remix-run/remix#4043]
### What version of Remix are you using?

@remix-run/express ^1.6.8

### Steps to Reproduce

1. Create a minimal Express-based app using `npx create-remix@latest` and the following settings:
```
? Where would you like to create your app? ./minimal
? What type of app do you want to create? Just the basics
? Where do you want to deploy? Choose Remix App Server if you're unsure; it's easy to change deployment targets. 
Express Server
? TypeScript or JavaScript? TypeScript
? Do you want me to run `npm install`? Yes
```
2. Run the app and visit in a browser
3. Open console to view errors

### Expected Behavior

No errors in browser console

### Actual Behavior

![image](https://user-images.githubusercontent.com/1980929/186068896-3f39aa27-6e5b-4375-9412-19eb88f7995d.png)

**Comment by rohanliston** [comment]
If I comment out `<Scripts />` in `root.tsx`, the errors disappear.

**Comment by machour** [comment]
This is one of your browser extensions injecting code : https://remix.run/docs/en/v1/pages/gotchas#browser-extensions-injecting-code

**Comment by rohanliston** [comment]
@machour Thanks for your help. I had looked at the page you linked; not sure how I missed that section! :sweat_smile: 

The extension causing the error was [Apollo Client DevTools](https://github.com/apollographql/apollo-client-devtools), which is something I have used for React development in the past. Is it something that should play nicely with Remix also? If so, should I submit an issue here or with the Apollo team (or both)?
